docker-compose.yml

```diff
@@ -72,6 +72,9 @@ services:
       - traefik.http.routers.appwrite_api_https.rule=PathPrefix(`/`)
       - traefik.http.routers.appwrite_api_https.service=appwrite_api
       - traefik.http.routers.appwrite_api_https.tls=true
+      - traefik.http.routers.appwrite_api_https.tls.certresolver=digitalocean
+      - traefik.http.routers.appwrite_api_https.tls.domains[0].main=$_APP_DOMAIN_FUNCTIONS
+      - traefik.http.routers.appwrite_api_https.tls.domains[0].sans=*.$_APP_DOMAIN_FUNCTIONS
```

Comment:
we can remove this also

Reply:
Removed digitalocean part, but rest is required even for self-hosted (to route requests properly)

Reply:
Removed all.